app/src/sharedTest/java/org/oppia/android/app/story/StoryFragmentTest.kt

```diff
@@ -135,6 +137,22 @@ class StoryFragmentTest {
     )
   }
 
+  private fun createTestStoryActivityIntent(): Intent {
+    return StoryActivity.createStoryActivityIntent(
+      ApplicationProvider.getApplicationContext(),
+      internalProfileId,
+      TEST_TOPIC_ID_1,
+      TEST_STORY_ID_1
+    )
+  }
+
+  private fun setStoryPartialProgressForFractions() {
```

Comment:
Can we shift this at the end of all the test, above `setUpTestApplicationComponent`, as it is a helper function.